packages/ui/src/composites/wui-button/index.ts

```diff
@@ -23,6 +23,8 @@ export class WuiButton extends LitElement {
 
   @property() public variant: 'accent' | 'fill' | 'shade' = 'fill'
 
+  @property() public onClick: () => void = () => null
```

Comment:
Can we add better type here, maybe `(event: MouseEvent) => void` or perhaps you can find even better html type for this, thats specific to buttons onclick